src/command/list/human.rs

```diff
@@ -661,17 +898,51 @@ See options for more detailed reports by running `volta list --help`.";
         use crate::command::list::{PackageManager, PackageManagerKind, Source};
 
         #[test]
-        fn none_installed() {
+        fn none_installed_npm() {
+            let expected = "⚡️ No npm versions installed.
```

Comment:
Should we change this message? It's not really true. Maybe we want something like "No custom npm versions installed" – it's just slightly strange to say *none* are given that you can totally type `npm <whatever>` and it uses the version from Node.

Note that changing it will definitely mean treating npm distinctly from Yarn—they won't be able to just share their whole implementation directly—but I think that's *probably* the right thing to do.

Reply:
That's a good call-out, I think we should change it to be more explicit about the difference between custom and bundled npm versions.